x-pack/plugins/ml/public/application/explorer/swimlane_container.tsx

```diff
@@ -245,7 +244,7 @@ export const SwimlaneContainer: FC<SwimlaneProps> = ({
     return isLoading
       ? containerHeightRef.current
       : // TODO update when elastic charts X label will be fixed
-        rowsCount * CELL_HEIGHT + LEGEND_HEIGHT + (true ? Y_AXIS_HEIGHT : 0);
+        rowsCount * CELL_HEIGHT + (true ? Y_AXIS_HEIGHT : 0);
```

Comment:
Is this causing issues in the embeddable, which now doesn't have enough height?

![image](https://user-images.githubusercontent.com/7405507/119814307-29511c80-bee2-11eb-9940-20b2d2cb8be2.png)

(Note on the view by swim lane on the left, only 9 rows are visible)

Reply:
Fixed in https://github.com/elastic/kibana/pull/99489/commits/344bce2d3a36738c647aae80812e3408db695f84

Reply:
Testing the latest edits (which fixed the earlier issue), I found another issue where there is no scrollbar inside the embeddable. The only way to see the other rows is to expand the height of the panel. This also occurs on latest master.

![image](https://user-images.githubusercontent.com/7405507/119854281-dc347100-bf08-11eb-86b0-4b632ae713f8.png)

Reply:
Raised https://github.com/elastic/kibana/issues/100865 for the issue with the embeddable swim lane container scrolling as it is unrelated to the changes in this PR.